### DISCUSSION TITLE
Allow photos in albums to be sorted by name, size or Custom order

### DISCUSSION THREAD

**Discussion by quietmansoath** [ente-io/ente#638] [Enhancements]
It looks like you can already sort albums by name, which is great. I don't see why there's no ability to sort the photos within the album by name.

I went through a lot of effort to rename some of my files that have their metadata kinda messed up for various reasons. But I have no way to easily recreate my sorting in the app versus in my desktop file viewer without custom or alphabetical sort.

Please consider adding this. I have thousands of photos I'll have to rename now (yet again) to get them to display properly in your app.

**Comment by alexpcooper** [comment]
Size, definitely. It'd be good to know which videos and images are using the most space, so I can decide upon whether they're worth keeping.

**Comment by silasdavis** [comment]
I know i have some multiple gigabyte videos that I do not want to keep, I really need to be able to do a descending sort by size to remove this outliers using up a disproportionate amount of my space

**Comment by Physed666** [comment]
PLEASE enable the Ente program to name each individual video / photo in a specific album

**Reply by silasdavis** [reply]
Not relevant to this ticket

**Reply by Physed666** [reply]
Thank you for NOT being nice or helpful.  
I do not appreciate ur lack of assistance on my issue / concern. 
I am very glad I am paying by the month and will cancel soon. If you are trying to lose new customers keep up the great service attitude you have demonstrated with me !

**Reply by silasdavis** [reply]
I'm just another customer not a representative of Ente. Since Ente is also an open source project this space GitHub is used to discuss issues in an open forum.

My response was a little curt because I am tracking this thread and was notified by email. Sorry about that.

**Comment by Physed666** [comment]
Since I am brand new to this and not a computer expert like yourself maybe instead of a reply like that you could assist me. Isnt that what this community is supposed to be about ? Must be nice to be that knowledgeable to give a flip reply which helps none in solving an issue I am having and attempting to resolve

**Reply by blackbird2150** [reply]
I'm new and just reading this, but for what it's worth you can do this now... If you click an asset > round circle with an " i " icon (**not** edit pencil icon) > description. When you go to that asset in the future description will pop up in the lower right. 
Ente support did say there was a bug that the description isn't always showing in the app, but does on browser, which is supposed to be fixed in a future release. 
Hope that helps. cheers.

**Comment by blackbird2150** [comment]
Adding support for custom ordering per album. 

Simply put albums are a story and I want to tell that story in the order I want. Specifically this is per album, not home screen view. 

Examples why this is helpful: food pics on a trip can be grouped together. Or for a cruise taking pics of the ship on different days and you want them together in the album. 

Sorting by size is also vital in a pay for storage model as called out by others.

**Reply by Arvi2026** [reply]
This is my most wanted feature and I am waiting fir it for long as I have hundreds of albums with custom order to import from google photos to ente and every new album i create always needs a custom irder of photos to tell a smooth story. Thanks for working on this!

**Comment by fnl** [comment]
Currently, the lack of custom sorting is a deal breaker for me and is why I am still sticking with Photos. Following this thread to figure out if things change.

**Reply by fnl** [reply]
Note that a simple “sort by name” (instead of date) would already make Ente usable to me.

**Reply by giacva** [reply]
> Note that a simple “sort by name” (instead of date) would already make Ente usable to me.

for me too. As soon as this option is available, I will switch to Ente."

**Comment by Poppi2** [comment]
They are a platform that provides a service to upload photo albums. How in gods name do they not have the basic of the most basic features a gallery could have. I literally do not grasp how a company could have put this little thought into designing their product. 
I can not even sort BY NAME...??

**Reply by Poppi2** [reply]
If your file names are in order but the creation date isnt, use this powershell command in the folder to change the metadata so the date matches the order of the file names. it is beyond sad one needs to use this work around but if anyone needs it, it is here. will be the last time i used this service.

```
$files = Get-ChildItem | Sort-Object Name
$time = Get-Date "2024-01-01 00:00:00"

foreach ($file in $files) {
    $file.CreationTime = $time
    $file.LastWriteTime = $time
    $time = $time.AddMinutes(1)
}
```
